com.unity.netcode.gameobjects/Editor/Configuration/NetcodeSettingsProvider.cs

```diff
@@ -40,6 +43,32 @@ private static void OnGuiHandler(string obj)
                 NetcodeForGameObjectsSettings.SetAutoAddNetworkObjectSetting(autoAddNetworkObjectSetting);
                 NetcodeForGameObjectsSettings.SetNetcodeInstallMultiplayerToolTips(multiplayerToolsTipStatus ? 0 : 1);
             }
+            const string generateNetworkPrefabsString = "Generate Default Network Prefabs List";
```

Comment:
You might think about migrating all of the prefab UI settings stuff within the same ditorGUI.BeginChangeCheck(); and ditorGUI.EndChangeCheck() region.  
All of the initialization inserted between lines 35 and 36.
Lines 60-66 inserted between 40-41.
Lines 69-70 inserted after line 44.

Reply:
Reworked this whole section. The display is nicer now. Got little collapsies and everything.

![image](https://user-images.githubusercontent.com/284434/203646031-e0c26f38-728d-486d-b7a6-f251dd527828.png)

Reply:
😻 ❤️‍🔥